gensim Quick Start.ipynb

```diff
@@ -56,7 +56,7 @@
    "source": [
     "## Accessing the `gensim` Jupyter notebooks\n",
     "\n",
-    "All of the `gensim` tutorials (including this document) are stored in [Jupyter notebooks](http://jupyter-notebook-beginner-guide.readthedocs.io/en/latest/what_is_jupyter.html).  These notebooks allow the user to run the code locally while working through the material.  If you would like to run a tutorial locally, first clone the GitHub repository for the project.\n",
+    "All of the `gensim` tutorials (including this document) are stored in [Jupyter notebooks](http://jupyter-notebook-beginner-guide.readthedocs.io/en/latest/what_is_jupyter).  These notebooks allow the user to run the code locally while working through the material.  If you would like to run a tutorial locally, first clone the GitHub repository for the project.\n",
```

Comment:
Ah
Accidentally removed the .html from this link instead of the unsupervised learning link